packages/mobile/src/web3/saga.test.ts

```diff
@@ -1,19 +1,13 @@
 import { expectSaga } from 'redux-saga-test-plan'
 import { call, delay, select } from 'redux-saga/effects'
 import { pincodeSelector } from 'src/account/reducer'
-import { waitForGethConnectivity } from 'src/geth/saga'
 import { navigateToError } from 'src/navigator/NavigationService'
+import { setLatestBlockNumber, updateWeb3SyncProgress } from 'src/web3/actions'
 import {
-  getLatestBlock,
-  setLatestBlockNumber,
-  setSyncProgress,
-  updateWeb3SyncProgress,
-} from 'src/web3/actions'
-import {
-  _CHECK_SYNC_PROGRESS_TIMEOUT,
-  _checkWeb3SyncProgressClaim,
-  checkWeb3Sync,
+  checkWeb3SyncProgress,
```

Comment:
@martinvol I do like the convention of `_` before functions only exported to test, but until this VSCode bug gets fixed, it just causes a headache :(
https://github.com/prettier/prettier-vscode/issues/716
We'd need to either disable VS code import organizing entirely (would not recommend) or fix this file manually before lint will pass.
Suggest we table this convention for now.

Reply:
Cool, no problem. Disabling VS code import organizing might not be a bad a idea since the pre-commit hook also sorts imports.

Reply:
Yeah but the way it de-dupes imports for you and removes unused ones is really nice